assets/blocks/reader-registration/style.scss

```diff
@@ -190,6 +225,27 @@
 	}
 }
 
+/* Sepcific styles to apply to .newspack-popup when Reader Registration is displayed */
```

Comment:
There's no conditional logic here – these styles will be applied to prompts regardless of their content:

<img width="828" alt="image" src="https://user-images.githubusercontent.com/7383192/188473366-e2146956-6ea4-41dc-ad1b-972c7852002b.png">